app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    OutputDocumentMailer.guidance_record(appointment_summary).deliver_later
```

Comment:
I'd be much happier with the mailer not having dependencies on other models. Named "value" parameters ftw. Potentially this means that we need the concept of a PDF?